README.md

```diff
@@ -193,3 +200,7 @@ This library is licensed under Apache 2.0. Full license text is available in
 ## Disclaimer
 
 This is not an official Google product.
+
+[logger-exporter]: https://www.rubydoc.info/gems/opencensus/OpenCensus/Trace/Exporters/Logger
+[stackdriver-exporter]: https://github.com/census-ecosystem/opencensus-ruby-exporter-stackdriver
+[zipkin-exporter]: https://github.com/census-ecosystem/opencensus-ruby-exporter-zipkin
```

Comment:
Is this accurate? https://github.com/census-ecosystem/opencensus-ruby-exporter-zipkin/blob/master/lib/opencensus/trace/exporters/zipkin.rb#L20 seems empty